src/Sylius/Bundle/InventoryBundle/Validator/Constraints/InStockValidator.php

```diff
@@ -22,6 +23,10 @@
 
 final class InStockValidator extends ConstraintValidator
 {
+    private const TARGET_CLASS = 'class';
+
+    private const TARGET_PROPERTY = 'property';
```

Comment:
Instead of those, the constants from `Symfony\Component\Validator\Constraint` should be used.